eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -39,8 +40,10 @@ jobs:
 
     ${{ if ne(parameters.displayName, '') }}:
       displayName: '${{ parameters.displayName }}'
+      name: ${{ replace(replace(parameters.displayName, ' ', '_'), '-', ' ') }}
```

Comment:
shouldn't we use parameters.jobName for this? it says it's required so should always be there

Reply:
The original code seems to let `displayName` override `jobName`.

Reply:
hm oh, I see what you mean. use jobName for the `name` here, since that will get used as the job name!